pom.xml

```diff
@@ -87,5 +92,10 @@
             <artifactId>mysql-connector-java</artifactId>
             <version>5.1.34</version>
         </dependency>
+        <dependency>
+	        <groupId>jline</groupId>
```

Comment:
what's this for? (the answer is: we want to do the client stuff properly - but it'd be nice if we could add a comment to that effect at very least)